geocity/apps/permits/api.py

```diff
@@ -288,7 +288,6 @@ class PermitRequestDetailsViewSet(
 
     throttle_scope = "permits_details"
     serializer_class = serializers.PermitRequestDetailsSerializer
-    permission_classes = [BlockRequesterUserWithoutGroup]
```

Comment:
Use `permission_classes = [BlockRequesterUserWithoutGroup]` instead of nothing.

**Why ?** 
This is an easier way to block all open routes directly from one location, if there is a security issue in geocity later.